src/app/story/story.component.spec.ts

```diff
@@ -55,7 +56,8 @@ describe('StoryComponent testing', () => {
             StoryService,
             DatePipe,
             {provide: ActivatedRoute, useValue: {params: routeParams}},
-            GreenwaveService
+            GreenwaveService,
+            { provide: Title, useClass: Title },
```

Comment:
Could this just be `Title,` instead of `{ provide: Title, useClass: Title },`? In my experience, `useClass` is used to override the default, such as when you need to mock the value. This isn't done here so I think it isn't necessary.